Configs/Debug.xcconfig

```diff
@@ -8,8 +8,8 @@
 
 #include "Shared.xcconfig"
 
-ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon
-CONFIGURATION = Debug
+ASSETCATALOG_COMPILER_APPICON_NAME = AppIconNightly
+CONFIGURATION = Nightly
```

Comment:
I am so not understanding this change. We want the nightly build to be a release build, yes? The debug builds are pretty much for developers. Developers need to be able to distinguish them from nightly builds.